src/debugpy/.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is this file supposed to be in source control? It might be, I just wasn't sure since it was in the `.vscode` directory

Reply:
It can be. It's common settings. It makes debugpy tests runnable from the test runner in VS code.